packages/next/src/server/lib/router-utils/setup-dev-bundler.ts

```diff
@@ -594,11 +599,18 @@ async function startWatcher(opts: SetupOpts) {
       ) as T
     }
 
+    type TurbopackMiddlewareManifest = MiddlewareManifest & {
+      instrumentation?: {
+        files: string[]
+        name: 'instrumentation'
+      }
+    }
```

Comment:
The per page manifest has this extra property. It's applied to all `functions` during merging, so it results in the same manifest as webpack